shuffle/cli/tutorials/HelloNFT.md

```diff
@@ -0,0 +1,154 @@
+Inside `nft/main/sources/nft`, create a new move file `MyNFT.move`.
+
+```
+module Sender::MyNFT {
+use Sender::NFTStandard;
+use Std::Signer;
+
+    struct MyNFT has drop, store {}
+
+}
+```
+
+## Add Minting Functionality
+
+Here we have imported `NFTStandard` and created a new NFT type `MyNFT`, which will be the `NFTType` we use when calling `NFTStandard` methods.
+
+```
+module Sender::MyNFT {
+use Sender::NFTStandard;
+use Std::Signer;
+
+    struct MyNFT has drop, store {}
+
+    public(script) fun mint_nft(account: signer, content_uri: vector<u8>) {
+        NFTStandard::initialize<MyNFT>(&account);
+        let token = MyNFT{};
+        let instance = NFTStandard::create<MyNFT>(
+            &account,
+            token,
+            content_uri,
+        );
+        NFTStandard::add(Signer::address_of(&account), instance);
+    }
+}
+```
+Now we add a wrapper for minting MyNFT, which 1) initializes the nft collection of type type `MyNFT`, 2) creates a `MyNFT` and 3) adds the created nft into the nft collection resource.
+
+## Publish
+Now that you have a very simple NFT module, you can publish it to your network:
+- Spin up your local node: `cargo run -p shuffle -- node`
+- Create an account for publishing: `cargo run -p shuffle -- account`
+- Deploy your module: `cd nft && cargo run -p shuffle -- deploy`
+
+If deployed successfully, you should MyNFT included in the list of published modules:
+
+```
+Skipping Module: 00000000000000000000000000000001::Hash
+Skipping Module: 00000000000000000000000000000001::Signer
+Skipping Module: 00000000000000000000000000000001::Vector
+Skipping Module: 00000000000000000000000000000001::Errors
+Skipping Module: 00000000000000000000000000000001::BitVector
+Skipping Module: 00000000000000000000000000000001::Capability
+Skipping Module: 00000000000000000000000000000001::FixedPoint32
+Skipping Module: 00000000000000000000000000000001::Option
+Skipping Module: 00000000000000000000000000000001::BCS
+Skipping Module: 00000000000000000000000000000001::GUID
+Skipping Module: 00000000000000000000000000000001::Event
+Deploying Module: 6215BB1C111E8943794F21EB7E559656::Message
+Deploying Module: 6215BB1C111E8943794F21EB7E559656::NFTStandard
+Deploying Module: 6215BB1C111E8943794F21EB7E559656::MyNFT
+Deploying Module: 6215BB1C111E8943794F21EB7E559656::TestNFT
+```
+
+## Explore MyNFT with Console
+You can now use shuffle console to interact with your MyNFT module! Start up your console within your project folder: `cargo run -p shuffle -- console`
+
+Let's try calling the `mint_nft` script function within the console.
+```
+> const contentUri = "https://placekitten.com/200/300"
+
+> let scriptFunction: string = context.defaultUserContext.address + "::MyNFT::mint_nft";
+
+> let typeArguments: string[] = [];
+
+> let args: any[] = [contentUri];
+
+> let txn = await helpers.invokeScriptFunction(scriptFunction, typeArguments, args);
+
+> txn = await devapi.waitForTransactionCompletion(txn.hash);
```

Comment:
As of last night, this method name has changed to `waitForTransaction` (no Complete)